owncloudComLibrary/src/main/java/com/owncloud/android/lib/common/utils/Log_OC.java

```diff
@@ -1,212 +1,52 @@
 package com.owncloud.android.lib.common.utils;
 
-import android.util.Log;
+import timber.log.Timber;
 
-import com.owncloud.android.lib.BuildConfig;
-
-import java.io.BufferedWriter;
 import java.io.File;
-import java.io.FileWriter;
-import java.io.IOException;
-import java.text.SimpleDateFormat;
-import java.util.Calendar;
-import java.util.Locale;
 
 public class Log_OC {
-    private static final String SIMPLE_DATE_FORMAT = "yyyy/MM/dd HH:mm:ss";
-    private static final String LOG_FOLDER_NAME = "log";
-    private static final long MAX_FILE_SIZE = 2000000; // 2MB
-
-    private static String mOwncloudDataFolderLog = "owncloud_log";
```

Comment:
we are losing this initialization, is it pointless?

Reply:
It's done in L45
`public static void startLogging(String storagePath)` and [here](https://github.com/owncloud/android/blob/6b989467ced4d47281afff37501cca77d5c0748d/owncloudApp/src/main/java/com/owncloud/android/MainApp.kt#L198-L200)